ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -1448,11 +1442,9 @@ public IXLDataValidation SetDataValidation()
             var validation = GetDataValidation();
             if (validation == null)
             {
-                using (var range = this.AsRange())
-                {
-                    validation = new XLDataValidation(range);
-                    Worksheet.DataValidations.Add(validation);
-                }
+                var range = this.AsRange();
```

Comment:
I introduced many of the `using` statements a few months back. If we're going to revert it, I'd prefer not to create `var range` variables, because they also weren't created prior to my changes. So I'd prefer code like this:

```c#
validation = new XLDataValidation(AsRange());
```

You should be able to see in the git log how they used to be. I'll help out with this change, because it's very trivial and a lot of work.

Reply:
I tried to do as you described but probably missed some of the occurrences.